src/AddLink.jsx

```diff
@@ -0,0 +1,19 @@
+import linkImg from './assets/link.svg';
+
+import './AddLink.css';
+
+const AddLink = () => {
+  return (
+    <div className='container-add-link'>
+      <div className='add-link'>
+        <div className='add-link_icon-box'>
+          <img src={linkImg} alt='링크'></img>
```

Comment:
self-closing은 신경써주시면 좋아요.
```suggestion
          <img src={linkImg} alt='링크' />
```